0007/README.md

```diff
@@ -0,0 +1,221 @@
+
+*   SC **no longer has 7 fixed seats**, but instead has a minimum of 3 and
+    ideally about 5.
+    *   Same guidelines for Maintainers, without the hard minimum.
+*   Individuals **self-nominate** to become a Maintainer or SC Member by
+    submitting a brief (1-2 paragraph) application consisting of:
+    *   **Evidence** of performing some of the role's duties over the recent
+        past.
+        *   For SC Member applications, this should include evidence of being
+            well-versed in SLSA.
+    *   **Statement of intent** to continue performing duties over the next
+        year.
+    *   **Affiliation** / employer.
+*   **SC Members decide** if the individual meets the desirable properties
+    (described in the next section) and respond with an official determination
+    and rationale. Decisions are effective immediately.
+    *   SC Members should attempt to reach consensus. At a minimum:
+        *   SC Member applications require explicit approval from at least
+            50% of the existing SC Members, excluding the candidate.
+        *   Maintainer applications require explicit approval from at least 2
+            existing SC members (possibly including the candidate).
+*   Existing SC Members and Maintainers **annually re-apply** for the position,
+    using the same lightweight process above.
+    *   Purpose: to provide a natural way to remove members who are no longer
+        actively performing their duties.
+    *   Exception: If there are only three SC Members, they remain indefinitely
+        until another person applies.
+*   Applications and decisions are **recorded publicly**, such as through
+    GitHub Issues, for transparency and to provide examples for future
+    applicants.
+*   **OpenSSF may**, at their discretion and with due process and input from
+    the SCI WG:
+    *   appoint new SC Members to get up to the minimum three; or
+    *   remove an SC Member for cause.
+* **Appeals** may be made to the SC directly, and if needed, escalated to the
+    OpenSSF.
+
+### Clarify project roles
+
+*   Participant: **no change**, but clarify language (it’s a bit confusing as
+    written)
+*   Contributor: **no change**, but clarify language
+    *   A Maintainer may **delegate** some duties to a Contributor, such as
+        permission to triage issues or approve PRs
+*   Maintainer: an individual who is **trusted** and **expected to regularly
+    perform** day-to-day duties **for a specific sub-project** within the
+    slsa-framework organization.
+    *   A **Core Maintainer** is a maintainer of the slsa-framework/slsa repo.
+        This repository is special because it contains the core specification
+        and the overall project issue tracker.
+    *   A **Subproject Maintainer** is a maintainer of a different repo within
+        the slsa-framework organization.
+*   SC Member: a **senior leader** of the SLSA project who is **trusted** and
```

Comment:
It may be nice for the SC group to present/review their activities quarterly / 6 month'ly at the standard SLSA meetings to improve transparency as well.

Reply:
I like the suggestion and agree with the principal. I am wary of codifying a periodicity, but added some text in [6a96a8c](https://github.com/slsa-framework/slsa-proposals/pull/15/commits/6a96a8cc37d3fe4d33321cacd63d49a58d6aae5e). WDYT?

Reply:
yes that update looks great.

Reply:
We want at least 2 different organizations represented, yes? Shouldn't that be clearly stated?